docs/install/installing-istio.md

```diff
@@ -91,7 +91,7 @@ spec:
               name: https
 EOF
 
-istioctl manifest apply -f istio-minimal-operator.yaml
+istioctl manifest install -f istio-minimal-operator.yaml --set values.gateways.istio-ingressgateway.runAsRoot=true
```

Comment:
Why do we need this?
Last time I installed Istio (1.6.x), it opens ports in the 1024+ range and the service maps the real port to it.
For example, port 80 is mapped to 8080.
Is this not the case in 1.5?

Reply:
I tested early in august, with version 1.6.x it worked (I prepared a demo for the GDG Cloud Melbourne). Recently, I tested the 1.7 version (for a demo to the GDG Nantes (a city in France), tonight: 6pm CEST) and it didn't work. I fixed like this. There might be a better/nicer way.

Reply:
better than runAsRoot is to modify the configuration so that the requested `targetPorts` are high enough not to require root. Modifying the `ports` section of Istio-minimal-operator.yaml to look like the below should work with 1.7:

~~~~
            - name: status-port
              port: 15021
              targetPort: 15021
            - name: http2
              port: 80
              targetPort: 8080
            - name: https
              port: 443
              targetPort: 8443
~~~~

Reply:
@julz, I committed a new version thank to your comment after having testing it!